source/scss/components/_messages.scss

```diff
@@ -0,0 +1,62 @@
+.fti-message {
+  color: $white;
+  padding: .875rem .9375rem;
+
+  &.info-message {
+    background-color: $cyan;
+  }
+
+  .message-container {
+    align-items: center;
+    display: flex;
+    min-height: 60px;
+
+    .icon-container {
+      align-items: center;
+      display: flex;
+      height: 24px;
+      justify-content: center;
+      width: 24px;
+
+      &.left-icon-container {
+        margin-right: 10px;
+      }
+
+      &.right-icon-container {
+        margin-left: 16px;
+      }
+    }
+
+    p {
+      font-size: 1rem;
```

Comment:
1rem is already having the root's font-size. Is there a parent element that affects this element's font-size? 
The same for line-height in the next line.